Tests/UnitTests/Plugins/Json/CMakeLists.txt

```
@@ -5,4 +5,5 @@ add_unittest(MaterialMapJsonConverter MaterialMapJsonConverterTests.cpp)
 add_unittest(AlgebraJsonConverter AlgebraJsonConverterTests.cpp)
 add_unittest(UtilitiesJsonConverter UtilitiesJsonConverterTests.cpp)
 add_unittest(SurfaceBoundsJsonConverter SurfaceBoundsJsonConverterTests.cpp)
-add_unittest(SurfaceJsonConverter SurfaceJsonConverterTests.cpp)
\ No newline at end of file
+add_unittest(SurfaceJsonConverter SurfaceJsonConverterTests.cpp)
+add_unittest(VolumeBoundsJsonConverter VolumeBoundsJsonConverterTests.cpp)
```

```suggestion
add_unittest(VolumeBoundsJsonConverter VolumeBoundsJsonConverterTests.cpp)
```
(adds new line at end of file)

---

added